src/app/main-nav/main-nav.component.html

```diff
@@ -1,29 +1,28 @@
 <mat-sidenav-container class="sidenav-container">
-  <mat-sidenav #drawer class="sidenav" fixedInViewport [attr.role]="(isHandset$ | async) ? 'dialog' : 'navigation'"
-    [mode]="(isHandset$ | async) ? 'over' : 'side'" [opened]="(isHandset$ | async) === false">
+  <mat-sidenav #drawer class="sidenav" mode="side" opened="true">
```

Comment:
Removing this causes issues with mobile layouts. What were you trying to do by removing it?

![angular-builders dev_home(iPhone 6_7_8)](https://user-images.githubusercontent.com/3506071/61978821-39cf9d80-afc0-11e9-8af8-aef3c0bc674f.png)